components/ide/jetbrains/backend-plugin/src/main/resources/META-INF/plugin.xml

```diff
@@ -22,18 +22,26 @@
     </dependencies>
```

Comment:
I have enabled auto-formatting of the code on save which made these adjustments. There are no functional changes otherwise.

Happy to revert, but I prefer following the suggestions of IntelliJ :)